### PR TITLE
#6: remove perl dependency and incorporate statusline output

### PR DESCRIPTION
**Pull request by els0r**
Covers the first two parts of #6 . I recommend to review and close this and tackle the HTTP server in a standalone issue.

This way, the changes from #8 can be merged into develop and we can go for a minor feature release.

### CODE REVIEW

File: cmd/goProbe/goProbe.go [hunk 1]
```diff
@@ -161,6 +163,7 @@ func main() {
 	}
 
 	// Open control socket
+	_ = os.RemoveAll(filepath.Join(dbpath, CONTROL_SOCKET))
```
Comment (fako1024) [inline cmd/goProbe/goProbe.go]: Error handling, even if it's a RemoveAll() ;-)

Comment (els0r) [reply cmd/goProbe/goProbe.go]: Fine... Even if this is opportunistic anyhow. What we shouldn't forget is to change this into a proper console log once #8 is merged.

**Comment by els0r** [comment]
@fako1024 : ping

File: cmd/goProbe/goProbe.go (hunk 1 above)
Comment (fako1024) [reply cmd/goProbe/goProbe.go]: +1 ! My concerns about the error logging is just born out of own fuckups, I recently traced a bug for two days just to figure out that I could have tackled it in 10 minutes had I just logged an error in an RemoveAll() statement ;-)

**Pull request merged**